src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -967,7 +967,7 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
   /**
    * When Done is clicked, stores updated addresses.
    */
-  protected async acceptChanges (): Promise<void> {
```

Comment:
I removed "protected" because VS Code was complaining that this method wasn't visible in the template (even though the code worked).